src/templates/addons/versionControl/index.ts

```diff
@@ -1,30 +1,63 @@
+import { prompt } from 'inquirer';
+
 import { GeneralOptions } from '@/commands/generate';
 import { copy } from '@/helpers/file';
 
-const versionControlChoices = [
-  {
-    type: 'list',
-    name: 'versionControl',
-    message: 'Which version control hosting would you like to use?',
-    choices: [
-      {
-        value: 'github',
-        name: 'GitHub',
-      },
+type VersionControlOptions = {
+  versionControlEnabled: boolean;
+  versionControlService: string;
+};
+
+const getVersionControlOptions = async (): Promise<VersionControlOptions> => {
+  const versionControlPrompt = await prompt([
+    {
+      type: 'confirm',
+      name: 'versionControlEnabled',
+      message: 'Would you like to enable version control?',
+      default: false,
+    },
+  ]);
+
+  if (versionControlPrompt.versionControlEnabled) {
+    const versionControlOptions = await prompt([
       {
-        value: 'none',
-        name: 'None',
+        type: 'list',
+        name: 'versionControlService',
+        message: 'Which version control hosting would you like to use?',
```

Comment:
```suggestion
        message: 'Which version control hosting would you like to use? [GitHub|Gitlab]',
```

Should we include the choices in the question so we know what to answer?

Reply:
Do we need it? The choices will be listed below the question:
![image](https://github.com/nimblehq/infrastructure-templates/assets/7344405/142239f5-9daa-4db0-8ded-528c0297fcc4)

Reply:
Sorry I missed that. perfect then!! Thanks